lms/static/scripts/frontend_apps/components/dashboard/DashboardActivityFilters.tsx

```diff
@@ -53,6 +54,19 @@ export type ActivityFilterItem<T extends Course | Assignment> = {
 export type DashboardActivityFiltersProps = {
   courses: ActivityFilterSelection | ActivityFilterItem<Course>;
   assignments: ActivityFilterSelection | ActivityFilterItem<Assignment>;
+
+  /**
+   * The segments filter is relevant only when listing students, so not
```

Comment:
If an API has a JSDoc comment, the first line should summarize what the field / method etc. is. Additional notes come after that in the next paragraph.